content/docs/install/_index.md

```diff
@@ -29,24 +29,40 @@ The steps to install and configure your Keptn instance are:
 
 5. [Authenticate CLI and Bridge](authenticate-cli-bridge)
 
-6. Install software to run Keptn tasks
-    * [Job Executor Service](https://github.com/keptn-contrib/job-executor-service/blob/main/docs/INSTALL.md)
-        * The [Job Executor Service](https://github.com/keptn-contrib/job-executor-service)
-         runs Keptn customizable tasks as Kubernetes jobs.
-         See [Job Executor Service Features](https://github.com/keptn-contrib/job-executor-service/blob/main/docs/FEATURES.md) for more details.
-        * Also see [Job Executor Service Architecture](https://github.com/keptn-contrib/job-executor-service/blob/main/docs/ARCHITECTURE.md#example-configuration)
-    * [Istio](istio)
-
-7. If you are using [Quality Gates](../concepts/quality_gates),
-   install the [monitoring service](monitoring) you want to use.
+6. If you have problems with your installation,
+   check out the hints in [Troubleshooting the installation](troubleshooting)
 
-8. Configure [webhooks](webhook_service) (optional)
 
-9. If you have problems with your installation,
-   check out the hints in [Troubleshooting the installation](troubleshooting)
+At this point, you should have a functional Keptn instance running.
 
 You may also be interested in the following topics:
 
 * [Upgrade Keptn](upgrade)
 
 * [Uninstall Keptn](uninstall)
+
+Additional software may need to be installed and configured
+to support the needs of the projects you are running:
+
+* Install software to run Keptn tasks
+    * [Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service)
+     that runs customizable Keptn tasks as Kubernetes jobs:
+        * Install the Integration on the Kubernetes cluster(s)
+        where the Keptn Execution Plane is installed.
+        * Follow the installation instructions for the integration.
+    * Install and configure [Istio](istio) on the Kubernetes cluster(s)
+    where the Keptn Execution Plane is installed.
+
+* If you are using [Quality Gates](../concepts/quality_gates),
+   install the [monitoring service](monitoring) you want to use
+   following the instructions for that Integration:
+   * DataDog
+   * [Dynatrace](https://artifacthub.io/packages/keptn/keptn-integrations/dynatrace-service)
+   * [Prometheus](https://artifacthub.io/packages/keptn/keptn-integrations/prometheus-service)
+   * To use a different Observability Platform as your data source,
+   follow the instructions in [Custom Integrations](http://localhost:1313/docs/0.17.x/integrations/)
```

Comment:
This links to localhost, might want a relative link

Reply:
blush!  Is fixed.

Reply:
fixed